tests/server/disco/TestViews.js

```diff
@@ -33,17 +33,27 @@ describe('Discovery Pane GET requests', () => {
       assert.include(policy.get('connect-src'), 'https://addons.mozilla.org');
     }));
 
-  it('should be using SRI for script and style in /', () => request(app)
+  it('should be using SRI for script and style', () => request(app)
     .get(defaultURL)
     .expect(200)
     .then((res) => checkSRI(res)));
 
-  it('should be a 404 for requests to /', () => request(app)
+  it('will redir to /en-US/firefox/  for requests to /', () => request(app)
```

Comment:
the CSP and redirect tests look overly repetitive to me since they are already covered elsewhere. I think this type of thing is better covered by checking that each app is configured with the correct units of logic. In other words, a test for each app that asserts the CSP configuration has been applied and that the prefixer middleware has been applied should be enough.